### DISCUSSION TITLE
Future Idea: Multiplication

### DISCUSSION THREAD

**Discussion by kathy-phet** [phetsims/number-line-operations#27] [Ideas]
In creating an explanation for a colleague, I thought about this extension to Number Line Operations that would address interpretation of the physical meaning of -2*-3. (e.g. removing -3 two times) . 

![image](https://user-images.githubusercontent.com/5096804/101022065-1c083600-352e-11eb-8164-88c9113cc2cd.png)

**Comment by amanda-phet** [comment]
Hey Kathy, I converted your issue to a "discussion" to see how this might work. What do you think of putting this idea here, instead of an "issue"? Let's try it out. If we don't like it, I'll put it back to an issue :) 

I like your idea here, since multiplication is often referred to as "repeated addition". But I think it would need a ton of thinking to make it work. I think someone could leverage the current sim (NLO) to address this concept without the explicit multiplication symbol. Students need to think of multiplication in other terms (grouping, arrays, etc.) way before they get to number lines, so by the time they get to number lines, this seems like it doesn't add much to that idea. I get that it can have negatives, but I'm also thinking about teachers who struggle with teaching multiplication of signed integers, and I doubt they would be able to use this without the legwork of (1) using a number line for addition of signed integers, and (2) describing multiplication as repeated addition. 

I'm not trying to say it's not worth exploring, but wanted to record my thoughts! I think we'd want to consider more than changing the operation control... 1–2x-3 is super complex. Maybe instead, the starting point is locked in at 0, so the expression can say -2x-3 instead. And there is a toggle between -2x-3 and --3--3. What would we do about 7x-3? Can we show -3-3-3-3-3-3-3?
